web/main.scss

```diff
@@ -0,0 +1,55 @@
+$title-section-background-color: #202124;
+$body-background-color: #E9ECEF;
+
+$title-font-color: rgba(0,0,0,.65);
+$regular-font-color: #212121;
+$parameter-font-color: #757575;
+
+body {
+  background-color: $body-background-color;
+}
+
+#title {
+  font-family: "Google Sans";
+  color: $title-font-color;
+}
+
+.section-name {
+  color: $regular-font-color;
+  font: 15px Roboto, sans-serif;
+  margin-bottom: 8px;
+}
+
+.card-header a:link, a:visited {
+  text-decoration:none;
+  color: $regular-font-color;
+  font: 16px Roboto, sans-serif;
+}
+
+.parameter-name {
+  color: $parameter-font-color;
+  font: 14px "Roboto Mono", monospace;
+}
+
+.parameter-input {
+  color: $regular-font-color;
+  font: 14px "Roboto Mono", monospace;
+}
+
+.underpacing-checkbox {
+  margin-top: 10px;
+  margin-bottom: 10px;;
+}
+.underpacing-checkbox-name {
+  font: 16px Roboto, "Helvetica Neue", sans-serif;
+}
+
+.btn.btn-success {
+  text-transform: uppercase;
+  font: 13px Roboto, sans-serif;;
+}
+
+.btn.btn-primary{
```

Comment:
try to use one font family for everything

Reply:
see [here](application)